aws/rust-runtime/aws-config/src/profile/credentials/repr.rs

```diff
@@ -123,15 +122,11 @@ pub(super) fn resolve_chain<'a>(
     // - There is not default profile
```

Comment:
I think this is a typo. It's not something you touched, so feel free to ignore this
```suggestion
    // - There is no default profile
```